README.md

```diff
@@ -167,9 +167,19 @@ The [CHANGELOG.md](/ably/ably-js/blob/main/CHANGELOG.md) contains details of the
 
 ---
 
-## Support, feedback and troubleshooting
+## Support and known issues
 
-Please visit https://support.ably.com/ for access to our knowledgebase and to ask for any assistance.
+For help or technical support, visit Ably's [support page](https://ably.com/support).
+
+
+## Unsupported features
```

Comment:
If the aim is to slim down the README then this information seems a bit niche; would this be better off on a documentation website?